src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Boot.ts

```diff
@@ -34,8 +36,28 @@ async function boot() {
     throw new Error(`Failed to start platform. Reason: ${ex}`);
   }
 
+  if (onBlazorInitializing) {
+    if (!window[onBlazorInitializing]) {
+      throw new Error(`Failed to find a function ${onBlazorInitializing} in 'window'`);
```

Comment:
Is there a strong why this needs to be an error? 

If there are being used to initialize the platform, maybe they should just be part of the platform? this feels like user-facing extensibility to me and I'm surprised that it would be an error to be unused.

Reply:
The way this works is as follows. You put something on your blazor-boot scrip tag like 
`on-blazor-intializing="myfunction"`
If we don't find it on the global namespace, we throw an exception to let you know that you told us to call your function but we could't find it.

Reply:
Oh yes that reminds me. As far as I can tell, these initialisation callbacks aren’t required by any other part of this PR. If that’s the case, could we remove them? Any new APIs of this kind would ideally be structured to account for some future changes to the bootstrapping process that we also have on the backlog.

Reply:
fair enough, I needed them in my original tests, and I think we need to look for a way to enable this for users and 3rd party libraries we inject, so that there's a flow where people can initialize blazor, register their own functions, boot the platform and execute functionality after the platform has booted.

My workaround was to set timeouts, which is why I added these.